src/components/Video/index.jsx

```diff
@@ -617,23 +617,39 @@ const Video = () => {
 							<p className="username-selector__label">
 								Set your username
 							</p>
-							<Input
-								placeholder="Username"
-								value={state.username}
-								onChange={(e) => handleUsername(e)}
-							/>
-							<Button
-								variant="contained"
-								color="primary"
-								onClick={connect}
-								className="username-selector__button"
-							>
-								Connect
-							</Button>
+							<div className="pair">
```

Comment:
Hey @ZohebMOPO , this looks good. However, the classes do not follow the BEM naming scheme.
Can you please change "pair" to something relevant? 
Here you can read more about BEM: http://getbem.com/introduction/